src/sgame/sg_bot_util.h

```diff
@@ -155,4 +156,10 @@ bool  BotPathIsWalkable( const gentity_t *self, botTarget_t target );
 //consider bot to be stuck if it does not move farther than this in some period of time
 constexpr float BOT_STUCK_RADIUS = 150;
 
+//Those are not really BG, but hopefully someday they will be.
+glm::vec3 BG_GetClientNormal( const playerState_t *ps );
```

Comment:
Do not write functions prefixed BG_ in sgame.

Reply:
This is fixed in a later commit. Not sure if you want me to cherry-pick even more commits in this PR or if you are ok with waiting on the next one.

Reply:
I think it's better to assume that a contributor may vanish at any moment. So no bad code should be merged with the promise of being fixed some time in the future.

Reply:
Ok but like, the code is already written.

So would you prefer more commits in this PR or a second one?

Reply:
Regardless of how you split it I prefer to have the commit that fixes it in the same one that messes it up.

Reply:
I've added a commit for that (to be squashed with everything else)